File: 0944-delete-columns-to-make-sorted/0944-delete-columns-to-make-sorted.py
class Solution:
    def minDeletionSize(self, strs: List[str]) -> int:
        n = len(strs[0])
        m = len(strs)
        count = 0
        for i in range(n):
            for j in range(m-1):
                if ord(strs[j][i]) > ord(strs[j+1][i]):
                    count += 1
                    break
        return count            

File: 0944-delete-columns-to-make-sorted/0944-delete-columns-to-make-sorted.cpp
class Solution {
public:
    int minDeletionSize(vector<string>& strs) {
        int n = strs[0].size();
        int m = strs.size();
        int res = 0;
        for(int i = 0; i < n; i++){
            for(int j = 0; j < m-1; j++){
                if((int)strs[j][i] > (int)strs[j+1][i]){
                    res++;
                    break;
                }
            }
        }
        return res;
    }
};